src/index.ts

```diff
@@ -170,6 +171,19 @@ export async function install(
 ) {
   const knownNamedExports = {...namedExports};
   const remotePackageMap = fromEntries(remotePackages);
+  const globDependencies = arrayOfDeps.filter(dep => glob.hasMagic(dep));
+  if (globDependencies.length) {
+    globDependencies.forEach(dep => {
+      const indexOfDep = arrayOfDeps.indexOf(dep);
+      arrayOfDeps.splice(indexOfDep, 1);
+    });
+    const globResults = globDependencies
+      .map(dep => path.join(cwd, 'node_modules', dep))
+      .map(dep => glob.sync(dep))
+      .flat()
```

Comment:
unfortunately we still need to support Node 8+, and I believe this was introduced in Node 11.

Here's another way you could write this, half getting rid of flat & half just doing some code golf. The nice thing about using a set is that now that we get deduplication for free (more important now that we have globs). But feel free to use this however you see fit.

```
const depsList = new Set();
arrayOfDeps.forEach(dep => {
  if (!glob.hasMagic(dep)) {
    depsList.add(dep);
    return;
  }
  const globLoc = path.join(cwd, 'node_modules', dep);
  glob.sync(globLoc).forEach((globMatchLoc) => {
    const globMatch = path.relative(`${cwd}/node_modules`, globMatch);
    depsList.add(globMatch);
  });
});
// Then, just replace the 1-2 later references to arrayOfDeps with the new depsList
```